### ISSUE TITLE
Feature Request: Disabling Workeer

### ISSUE THREAD

**Issue by FexMa** [d0sboots/PerfectTower#4]
Please add to it that this script: Disables the "adding items to the dissolv queue" Worker on production start.
So its not Producing things and instantly Disolving them when the worker is active

**Comment by d0sboots** [comment]
Yup, it was already on my list of things to add now that worker support is added to the AI, but it's good to track it here as well.

**Comment by FexMa** [comment]
Thats nice to hear thanks for your amazing work

**Comment by d0sboots** [comment]
Oops, forgot to close this out. I implemented this... a while ago now. (But the dissolve worker has to be assigned to the factory worker slot.)
